tests/test_check.py

```diff
@@ -256,6 +265,50 @@ def test_check_no_reporter_output(
         remove("pyta_output.html")
 
 
+@pytest.mark.parametrize("input_file", INPUTS["test_check_error_raise"])
+def test_check_error_raise(input_file: str | list[str]) -> None:
+    """Test that setting on_verify_fail='raise' causes check_all to raise an error for invalid input files."""
+    with pytest.raises(ValueError, match="File cannot be checked"):
+        python_ta.check_all(
+            input_file,
+            config={
+                "output-format": "pyta-plain",
+                "pyta-error-permission": "no",
```

Comment:
It's not necessary to add `pyta-error-permission`/`pyta-file-permission`, as the default values are `"no"`. I think other tests containing them are a holdover from previous behaviour.